Tests/CommandsTests/PackageCommandTests.swift

```diff
@@ -2146,6 +2146,76 @@ class PackageCommandTestCase: CommandsBuildProviderTestCase {
                 )
             }
         }
+
+        try await fixture(name: "SwiftMigrate/StrictMemorySafetyMigration") { fixturePath in
```

Comment:
Was there a reason to not define a function over the fixture and feature names for these?

Reply:
Laziness. I'll refactor it